### ISSUE TITLE
Long delayed CV info can let mylar mark issues as skipped

### ISSUE THREAD

**Issue by SenorSmartyPants** [mylar3/mylar3#709]
**Describe the bug**
If CV hasn't posted info on new issues by the sunday or monday following release, mylar can sometimes mark new issues as skipped when info is finally available. This has happened several times with Marvel issues.

**Environment (please complete the following information):**
 - OS: Docker hotio/mylar3:nightly
 - Branch / commit: [4252af4540cefad00c8330c33bf91c4ae76a6485]

**Comment by evilhero** [comment]
We're aware of some of the problems that has been occurring with the late updates from CV - I added in some extra checks along with extending the refresh length , so aside from waiting to see how things pan out when they don't get updated, it's kind of hard to test otherwise.

Just keep us posted if you notice something odd from lack of updating due to the switch to the following week.

**Comment by SenorSmartyPants** [comment]
Late saturday? Marvel issues got into CV. I think I saw the issues get set to skipped initially, but then they were downloaded today.

**Comment by evilhero** [comment]
Makes sense - when the weeklypull refreshes for the current week (either automatically via the scheduled job, or if manually recreating the pull) it only checks the current week. So the pull doesn't update weeks that are not the current week unless you manually recreate that particular week. 

So they would've remained Skipped as the date would've passed for the cutoff, but when the db updater runs at the scheduled 24hr interval it would have caught those recent updates, and then updated the required series so that it shows the new information and then it would've gotten marked as Wanted. Now somewhere in here there lies the possible problem that you've eluded to about issues getting set to Skipped and not coming out of that status when the week rolls over. 

There were also some major changes / updates that went into the python3-dev branch over the last few weeks that may have fixed most of these problems.

In addition, I just did a PR that will requery the previous week when it updates the weekly pull for the current week only (either via scheduled job, or manually recreating the pull). That should be able to snag any outliers, in addition to the db update change increasing the chances that everything is updated accordingly as CV updates things.

**Comment by SenorSmartyPants** [comment]
Mylar has been grabbing stuff on Sunday mornings after the week change with no problems lately.
